src/acom_music_box/music_box.py

```diff
@@ -68,7 +68,6 @@ def solve(self, output_path=None, callback=None):
         the specified file.
 
         Args:
-            output_path (str, optional): The path to the file where the output will be written. If None, no output file is created. Defaults to None.
             callback (function, optional): A callback function that is called after each time step. Defaults to None. The callback will take the most recent results, the current time, conditions, and the total simulation time as arguments.
```

Comment:
Could you make multiple lines instead of one long line for the description? This will help make it easier to read.